gitgud/skills/user_messages.py

```diff
@@ -177,3 +184,10 @@ def display_entry(index, human_name, code_name, indent):
                 code_name=item.name,
                 indent=indent
             )
+
+
+@separated
+def firstcommit_status(created, added, committed):
```

Comment:
See other comment, I think we can have this function be a part of the `FirstCommit` class rather than separate